src/components/inputs/select-input/select-input.spec.js

```diff
@@ -215,20 +215,13 @@ describe('in multi mode', () => {
         expect(onChange).not.toHaveBeenCalled();
         expect(queryByText('Mango')).toBeInTheDocument();
       });
-      it('should call not call onChange when value is cleared by clicking delete button ', () => {
-        const onChange = jest.fn();
-        const { getByLabelText, getByText, queryByText } = renderInput({
-          onChange,
+      it('should not render the clear button', () => {
+        const { queryByTitle } = renderInput({
           isMulti: true,
           value: ['mango'],
           isDisabled: true,
         });
-        const input = getByLabelText('Fruit');
-        fireEvent.focus(input);
-        const deleteButton = getByText('Mango').nextSibling;
-        deleteButton.click();
-        expect(onChange).not.toHaveBeenCalled();
```

Comment:
These were giving a false positive: onChange wasn't called because the ClearIndicator isn't rendered when the component is disabled, so clicking this "nextSibling" wouldn't do anything anyway.